src/WebJobs.Script.WebHost/Diagnostics/EventGenerator.cs

```diff
@@ -75,7 +251,7 @@ public void LogFunctionDetailsEvent(string siteName, string functionName, string
             {
                 if (IsEnabled())
                 {
-                    WriteEvent(57908, siteName, functionName, inputBindings, outputBindings, scriptType, isDisabled);
+                    WriteEvent(59708, siteName, functionName, inputBindings, outputBindings, scriptType, isDisabled);
```

Comment:
Oh man thank you github ui. Stared at this way too long in gitk trying to figure out what had changed, ended up assuming it was whitespace.

Definitely do not want/did not intend to change this id